### PR TITLE
adding dashboard, fixing issue #287

### PR DESCRIPTION
**Pull request by hTrap**
fixing issue #287 , also added dashboard to the dropdown. The app name profile wasn't available due to some python conflict, so named the app profile. The UI currently looks like this
dash

If further changes in the dashboard template is required, kindly let me know.

### CODE REVIEW

File: junction/urls.py [hunk 1]
```diff
@@ -47,6 +47,9 @@
 
     url(r'^api/v1/', include(router.urls)),
 
+    # User Dashboard
+    url(r'^dashboard/', include('junction.profiles.urls')),
```
Comment (kracekumar) [inline junction/urls.py]: Make the url `/profiles/`.'

File: junction/profiles/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^$', views.dashboard, name='dashboard'),
```
Comment (kracekumar) [inline junction/profiles/urls.py]: Make it 'dashboard$` as pattern.

File: junction/profiles/views.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+from django.contrib.auth.decorators import login_required
+from django.views.decorators.http import require_http_methods
+
+
+@login_required
+@require_http_methods(['GET'])
+def dashboard(request):
+    proposal_qs = Proposal.objects.order_by('conference__end_date')
+    user_proposal_list = []
+    if request.user.is_authenticated():
```
Comment (kracekumar) [inline junction/profiles/views.py]: This is repetitive, decorator `@login_required` takes care of auth. You can remove this check.

We need to display proposals based on conference.
View will look like
```
conferences = Conference.objects.order_by('end_date')
return render(request, 'profiles/dashboard.html', {'conferences': conferences})
```

File: junction/templates/profiles/dashboard.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        <a href="#" class="navbar-brand">Dashboard</a>
+{% endblock navbar_logo %}
+
+
+{% block content %}
+<section class="content custom-container proposal-list">
+    <div class="push-4-bottom push-1-top">
+        <div class="row">
+            <h2>Your Proposals</h2>
+            {% if user_proposal_list %}
```
Comment (kracekumar) [inline junction/templates/profiles/dashboard.html]: Display `conf_proposals` . Proposal should be displayed under each conf name.

**Comment by hTrap** [comment]
made the said changes, though a bit ambiguous about the url (current is profiles/), please do suggest any other changes.

teste1

**Pull request merged**